src/api/comments.ts

```diff
@@ -1,8 +1,7 @@
-// import { client } from '../utils/axiosClient';
 import { client } from '../utils/fetchClient';
 import { Comment } from '../types/Comment';
 
-export const getPostComments = (postId: number) => {
+export const getPostComments = (postId: number | undefined) => {
```

Comment:
Better check if value is undefined on upper level and just don't call a function if it is.